libs/ui/tooltip/brain/src/lib/brn-tooltip-content.component.ts

```diff
@@ -60,8 +65,7 @@ export class BrnTooltipContentComponent implements OnDestroy {
 	public readonly _tooltipClasses = signal('');
 	public readonly side = signal('above');
 	/** Message to display in the tooltip */
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	public template: TemplateRef<any> | null = null;
+	public message: string | TemplateRef<unknown> | null = null;
```

Comment:
Maybe we should name this content instead of message?

Reply:
Renamed